.eslintrc.js

```diff
@@ -21,33 +23,52 @@ module.exports = {
     '**/ios/main.jsbundle',
     '**/stylelint.config.js',
     '**/dist/',
-    '**/lib-dist/'
+    '**/lib-dist/',
+    '.eslintrc.js'
   ],
   rules: {
-    'arrow-body-style': 'error',
-    'arrow-parens': ['error', 'as-needed'],
+    // Overly strict rules (for now)
+    'class-methods-use-this': 'off',
+    'default-case': 'off',
+    'global-require': 'off',
+    'import/extensions': 'off',
+    'import/no-extraneous-dependencies': 'off',
+    'import/no-unresolved': 'off',
+    'import/prefer-default-export': 'off',
+    'lines-between-class-members': 'off',
+    'no-shadow': 'off',
+    'no-nested-ternary': 'off',
+    'no-underscore-dangle': 'off',
+    'no-use-before-define': 'off',
```

Comment:
Why not?

Reply:
We already have `'@typescript-eslint/no-use-before-define': 'error'` which does the same with typescript support. This rule also throws the error all the time:
> 'React' was used before it was defined  no-use-before-define